src/main/webapp/app/entities/symptom/symptom-detail.component.ts

```diff
@@ -1,24 +1,52 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { Subscription } from 'rxjs';
+import { JhiEventManager } from 'ng-jhipster';
 
 import { ISymptom } from 'app/shared/model/symptom.model';
+import { SymptomService } from './symptom.service';
+import { SymptomDeleteDialogComponent } from 'app/entities/symptom/symptom-delete-dialog.component';
 
 @Component({
   selector: 'custom-symptom-detail',
   templateUrl: './symptom-detail.component.html',
 })
 export class SymptomDetailComponent implements OnInit {
   symptom: ISymptom | null = null;
+  eventSubscriber?: Subscription;
 
-  constructor(protected activatedRoute: ActivatedRoute) {}
+  constructor(
+    protected symptomService: SymptomService,
+    protected activatedRoute: ActivatedRoute,
+    protected modalService: NgbModal,
+    protected eventManager: JhiEventManager,
+   ) {}
 
   ngOnInit(): void {
     this.activatedRoute.data.subscribe(({ symptom }) => {
       this.symptom = symptom;
     });
+    this.registerChangeInSymptom();
+  }
+
+  ngOnDestroy(): void {
+    if (this.eventSubscriber) {
```

Comment:
There seems to be some github actions that you need to deal with. Please check this.